sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_id_documents_async.py

```diff
@@ -0,0 +1,90 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_recognize_id_documents_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to recognize fields from an identity document.
+
+    See fields found on an ID document here:
+    https://aka.ms/formrecognizer/TODO
+
+USAGE:
+    python sample_recognize_id_documents_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer API key
+"""
+
+import os
+import asyncio
+
+
+class RecognizeIdDocumentsSampleAsync(object):
+
+    async def recognize_id_document(self):
+        path_to_sample_forms = os.path.abspath(os.path.join(os.path.abspath(__file__),
+                                                            "..", "./../sample_forms/id_documents/license.jpg"))
+
+        # [START recognize_id_document]
```

Comment:
to match the end clause
```suggestion
        # [START recognize_id_documents]
```

Reply:
I'd been going back and forth about having the `s` at the end. Do you think I should rename the functions to have an `s` at the end?

Reply:
Fixed. I decided to add the `s` to the function as well.